modules/jpa/src/main/java/io/oasp/module/jpa/dataaccess/api/DatabaseConfigProperties.java

```diff
@@ -10,36 +13,33 @@
  *
  * @since 3.0.0
  */
-class DatabaseWorkaround {
+class DatabaseConfigProperties {
 
-  private static final Logger LOG = LoggerFactory.getLogger(DatabaseWorkaround.class);
+  private static final Logger LOG = LoggerFactory.getLogger(DatabaseConfigProperties.class);
 
-  private static DatabaseWorkaround instance;
+  private static DatabaseConfigProperties instance;
 
-  private final int maxSizeOfInClause;
+  @Value("${database.query.in-clause.max-values:2147483647}")
+  private int maxSizeOfInClause;
 
   /** Die maximale Anzahl von Werten, welche Oracle in einer IN-Klausel verarbeiten kann. */
   public static final int MAX_SIZE_OF_IN_CLAUSE_IN_ORACLE = 1000;
```

Comment:
I think we do not need this field anymore as we cannot set it programmatically on runtime now.
Maybe worth to add it commented to a basic application.properties as well as the property itself also commented.

Reply:
I do not see why we do not need this field anymore. Would you suggest rather to inject the entire spring boot `application.properties` as a `Map` or `Environment` instead and get it manually from there? IMHO using the `@Value` is more compact, more expressive and simpler. Also it is the suggested way to do it according to the spring documentation.

I do agree that we need some kind of documentation for this feature. Once this PR is agreed and merged, I would also provide documentation for the entire feature in the forge wiki. We could also add a comment to the `application.properties` of our archetype but as we have so many PRs open at the moment I would prefer to get things done and merged first. For nice-to-haves we should create issues and/or follow up PRs. Currently I feel like being in a kind of cage to get fast progress and improvements into OASP4J. Also the PR #629  for our archetype was a nightmare and turned again into a monster PR with tons of different aspects. However, we also fail to get PRs reviewed and merged quickly so in such cases it would even take way longer to create small PRs as I would need >10 PRs then and wait weeks up to moths for each to get merged while I am blocked for the follow up PR as it depends on the previous PR to be done.